src/main/java/com/stage/users/Controllers/UserController.java

```diff
@@ -0,0 +1,83 @@
+package com.stage.users.Controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.stage.users.Iservice.UserIservice;
+import com.stage.users.Model.User;
+
+
+@RestController
+@RequestMapping("users")
+public class UserController {
+	
+	@Autowired
+	UserIservice userservice;
+	
+	
+	@PostMapping("/adduser")
```

Comment:
the url would be ......./users/adduser : do you think this is a convenient naming ?